exe/kubernetes-deploy-restart

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+require 'kubernetes-deploy'
+require 'kubernetes-deploy/restart_task'
+
+raw_deployments = nil
+ARGV.options do |opts|
+  opts.on("--deployments=LIST") { |v| raw_deployments = v }
+  opts.parse!
+end
+
+if raw_deployments.nil?
+  puts "Failed: specify at least one deployment to restart with --deployments flag"
```

Comment:
And related to this, could this task use `KubernetesDeploy.logger` for output consistency with regular deploys, and sensitivity to `ENV["LEVEL"]` and `ENV["DEBUG"]`? It's in `lib/kubernetes-deploy.rb`, so it's actually already require'd.